File: 119/py/119.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        result = [[1]]

        for i in range(1, rowIndex+1):
            newR = []
            for j in range(0, i+1):
                if j == 0 or j == i:
                    newR.append(1)
                else:
                    newR.append(result[i-1][j] + result[i-1][j-1])

            result.append(newR)
        
        return result[rowIndex]

File: 119/js/119.js
/**
 * @param {number} rowIndex
 * @return {number[]}
 */
var getRow = function(rowIndex) {
    let result = [[1]];

    for (let i = 1; i <= rowIndex; i++ ){
        let newR = [];
        for (let j = 0; j < i+1; j++){
            if (j === 0 || j === i){
                newR[j] = 1;
            }else{
                newR[j] = result[i-1][j] + result[i-1][j-1];
            }
        }
        result[i] = newR;
    }
    return result[rowIndex];
};